pandas/tests/frame/test_duplicates.py

```diff
@@ -182,6 +182,12 @@ def test_drop_duplicates():
         assert df.duplicated(keep=keep).sum() == 0
 
 
+def test_duplicated_on_empty_frame_gives_back_frame():
```

Comment:
can you add the issue number as a comment here, rename the test to

test_duplicated_on_empty_frame